### ISSUE TITLE
Control.Monad.Error.Lens

### ISSUE THREAD

**Issue by tonymorris** [qfpl/papa#5]
The `Control.Monad.Error.Lens` module should be in `module Papa`.

**Comment by tonymorris** [comment]
@puffnfresh Do you use this module?

**Comment by puffnfresh** [comment]
@tonymorris no, I don't but it conflicts with `Control.Exception.Lens` which seems pretty useful though I've only used it a tiny amount.

**Comment by tonymorris** [comment]
OK yeah, maybe that is the right module to export.

**Comment by tonymorris** [comment]
`Control.Exception.Lens` seems more geared toward `Exception e =>` rather than `MonadError e m =>`. 

I prefer `MonadError` since it corresponds to exactly `MonadEither` if it were to exist (despite the word error in the name). I think `Exception` is busted.

**Comment by puffnfresh** [comment]
I think `Control.Exception.Lens` is useful but probably for legacy reasons. I agree it doesn't make much sense in a replacement Prelude.
